app/views/user_added_to_project_mailer/added_to_project.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>You have been added to the Zooniverse project <%= @project.display_name %> in the following roles: <%= @roles.join(" and ") %></p>
+
+<p>Visit <a href="https://zooniverse.org/projects/<%= @project.slug %>"><%= @project.display_name %></a> to get started.</p>
+
+<% if @roles.include? "collaborator" %>
+  <p>To edit your project, go to <a href="https://zooniverse.org/lab/<%= @project.id %>">
```

Comment:
anywhere you've got the hardcoded URL we should be looking to use the frontend URL config instead.